### ISSUE TITLE
`scdel` should allow "deletion" of objects and group targets

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-R/scipiper#98]
See also issue #15 (warnings on `scdel(object)`) and issues encountered in getting vignettes+tests to pass in PR #86 (e.g., https://travis-ci.org/USGS-R/scipiper/jobs/488817085#L1203)

**Comment by aappling-usgs** [comment]
#15 was resolved sometime before today, and remake:::delete() seems to work fine for objects.

Group (fake) targets are never current and have no entries in the remake database, as far as I can tell. Therefore scdel just shouldn't bother attempting to `remake:::delete()` them. This is resolved by 19fc637

**Comment by aappling-usgs** [comment]
Evidence that #120 fixes this issue:

example remake.yml:
```yml
targets:
  wqp_pull_string:
    command: c(I('hi mom!'))
  
  fake_target:
    depends:
      - wqp_pull_string
```

commands to build and delete the object and group=fake targets:
```r
> scmake('wqp_pull_string')
Starting build at 2019-08-30 17:16:32
<  MAKE > wqp_pull_string
[ BUILD ] wqp_pull_string                                  |  wqp_pull_string <- c("hi mom!")
Finished build at 2019-08-30 17:16:32
Build completed in 0.00 minutes
> scmake('fake_target')
Starting build at 2019-08-30 17:16:36
<  MAKE > fake_target
[    OK ] wqp_pull_string
[ ----- ] fake_target
Finished build at 2019-08-30 17:16:36
Build completed in 0.00 minutes
> scdel('wqp_pull_string')
(   DEL ) wqp_pull_string                                  |  rm("wqp_pull_string")
> scdel('fake_target')
> scmake('fake_target')
Starting build at 2019-08-30 17:16:45
<  MAKE > fake_target
[ BUILD ] wqp_pull_string                                  |  wqp_pull_string <- c("hi mom!")
[ ----- ] fake_target
Finished build at 2019-08-30 17:16:45
Build completed in 0.00 minutes
> scdel('fake_target')
> scdel('wqp_pull_string')
(   DEL ) wqp_pull_string                                  |  rm("wqp_pull_string")
> scdel('wqp_pull_string')
(       ) wqp_pull_string
```
(note that scdel of a fake target does nothing, which is pretty much consistent with remake's refusal to delete fake targets.)
